index.ts

```diff
@@ -181,7 +181,7 @@ export default class Analytics {
       looselyValidate(message, type);
     } catch (e) {
       if (e.message === 'Your message must be < 32kb.') {
```

Comment:
after testing, I've confirmed that events larger than 32 kb still get processed by the rudder-server. This is contrary to what the [docs](https://docs.rudderstack.com/rudderstack-api/http-api#4-maximum-allowed-request-size) state. I assume it's because we're self-hosting. For this reason, I didn't make this check stricter (events larger than 32kb still get enqueue'd and sent off)